test/drenv/envfile_test.py

```diff
@@ -48,9 +46,19 @@ def valid_env(tmpdir):
     memory: 8g
   - name: dr2
     template: dr-cluster
+    # These are specific values and will be propagated to minikube as is.
+    driver: myhypervisor
+    network: mynetwork
   - name: hub
     external: true
     template: hub-cluster
+    # Using driver and network from platform defaults 
+    driver: $container
+    network: $network
+  - name: dr3
+    template: dr-cluster
+    # Using driver from platform defaults 
```

Comment:
Same here.

I'm using these vim setting to highlight trailing whitespace:

```
highlight ExtraWhitespace ctermbg=red guibg=red                                                                               
match ExtraWhitespace /\s\+$/
autocmd BufWinEnter * match ExtraWhitespace /\s\+$/
autocmd InsertEnter * match ExtraWhitespace /\s\+\%#\@<!$/
autocmd InsertLeave * match ExtraWhitespace /\s\+$/
autocmd BufWinLeave * call clearmatches()
```

Reply:
`git diff` also display the trailing whitepsace:

![Screenshot from 2023-07-19 00-24-29](https://github.com/RamenDR/ramen/assets/395026/7441ec76-22ad-4d93-9dcd-b5b2679c700b)